.ci/pull-request-native-config.yaml

```
@@ -0,0 +1,57 @@
+version: "2.1"
+
+dependencies: ./project-dependencies-quarkus.yaml
```

No quarkus dependency for native check

```suggestion
dependencies: ./project-dependencies.yaml
```